utils/manifests/generateComponent.go

```diff
@@ -0,0 +1,100 @@
+package manifests
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"github.com/layer5io/meshkit/utils"
+)
+
+func generateComponents(manifest string, resource int, cfg Config) (*Component, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+
+	var binPath string = wd + "/utils/manifests/utilBin/kubeopenapi-jsonschema"
```

Comment:
A good place to store the fetched binary would be in `~/.meshery/bin` which will ensure multiple components can use it.

Reply:
@DelusionalOptimist  yups, didnt think of that